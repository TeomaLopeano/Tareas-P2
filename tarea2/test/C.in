estaDefinidaActual
agregarAIterador 5
reiniciarIterador
estaDefinidaActual
actualEnIterador
avanzarIterador
estaDefinidaActual
agregarAIterador 6
agregarAIterador 7
agregarAIterador 8
reiniciarIterador
actualEnIterador
avanzarIterador
avanzarIterador
avanzarIterador
estaDefinidaActual
actualEnIterador
avanzarIterador
estaDefinidaActual
reiniciarIterador
avanzarIterador
actualEnIterador
Fin