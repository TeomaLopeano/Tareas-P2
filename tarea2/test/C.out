1>Actual de it NO está definida.
2>Agregando 5 a it.
3>Reiniciando it.
4>Actual de it está definida.
5>El actual de it es 5.
6>Avanzando en it.
7>Actual de it NO está definida.
8>Agregando 6 a it.
9>Agregando 7 a it.
10>Agregando 8 a it.
11>Reiniciando it.
12>El actual de it es 5.
13>Avanzando en it.
14>Avanzando en it.
15>Avanzando en it.
16>Actual de it está definida.
17>El actual de it es 8.
18>Avanzando en it.
19>Actual de it NO está definida.
20>Reiniciando it.
21>Avanzando en it.
22>El actual de it es 6.
23>Fin.
